src/contributor-guide/pull-request-tests.md

```diff
@@ -0,0 +1,131 @@
+---
+group: contributor-guide
+title: Pull Request Tests
+---
+
+The following topics are included in this guide:
+
+-  [Magento Automated Testing](#magento-automated-testing)
+-  [Running tests for a pull request](#run-pull-request-tests)
+-  [Related pull requests](#related-pull-requests)
+
+## Magento Automated Testing {#magento-automated-testing}
+
+Magento Automated Testing is a testing infrastructure that allows to run the set of all required Magento Automated Tests against code in Pull Requests. It can be triggered via a comment in a pull request. The running tests are represented both as Pull Request statuses and as Pull Requests checks.
+
+Below there's information about all kinds of tests available currently.
+
+### Semantic Version Checker
+
+Semantic Version Checker is a test that compares the changed code against the mainline and detects levels of code changes. It shows a list of changes grouped by the class (level) of changes (MAJOR, MINOR, PATCH) with explanation of the change (Class was removed, [public] Method implementation changed, [private] Property has been removed etc.) The allowed level is PATCH.
+Please see the [Backward Compatibility Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/index.html) for more information
+
+### Functional Tests
+Functional Tests are PHPUnit-based tests that check Magento functional behavior from the perspective of a user. They press buttons and links in browser, fill and submit forms, look and check what is displayed in the browser's window like real users.
```

Comment:
Here there were added static checks of MFTF tests. Please talk to Alex Kolesnyk

Reply:
@okolesnyk can you please help to update the content of documentation?

Reply:
@slavvka do you think we can go with general tests description, not listing all the subtypes of tests for now?